ddtrace/tracer/option.go

```diff
@@ -159,8 +159,14 @@ func WithPropagator(p Propagator) StartOption {
 	}
 }
 
-// WithServiceName sets the default service name to be used with the tracer.
+// WithServiceName sets the default service name to be used with the tracer. It will be
+// deprecated in favour of WithService in the next major version.
```

Comment:
Nits:
1. Deprecated != removed.
2. We might not actually remove it so maybe we shouldn't say so. Your call :-)

Reply:
You're right. I changed the text.